src/node_interface.rs

```diff
@@ -20,6 +20,15 @@ use name_type::NameType;
 use types::{Authority, DestinationAddress};
 use super::{Action, RoutingError};
 
+pub enum RoutingNodeAction {
```

Comment:
in lib.rs we have an `Action` object: 
```pub enum Action {
  Reply(Vec<u8>),
  SendOn(Vec<NameType>),
}```

Should this be taken into consideration?